### PR TITLE
[hi] Add docs/tasks/tools/included/_index.md

### PR DESCRIPTION
**Pull request by kaiwalyakoparkar**
Localizes
- `content/hi/docs/tasks/tools/included/_index.md`

### CODE REVIEW

**Comment by kaiwalyakoparkar** [comment]
This PR is in continuation of hi localization effort: #29353

/language hi

File: content/hi/docs/tasks/tools/included/_index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: "उपकरण शामिल"
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/_index.md:2]:
```suggestion
title: "शामिल उपकरण"
```

File: content/hi/docs/tasks/tools/included/_index.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+description: "Snippets to be included in the main kubectl-installs-*.md pages."
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/_index.md:3]:
```suggestion
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

**Comment by anubha-v-ardhan** [comment]
@kaiwalyakoparkar Thanks for contributing : )
Please sign the CLA so your PR can reviewed 
https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by kaiwalyakoparkar** [comment]
@anubha-v-ardhan I have signed the cla. It still shows that not passed. What should I do? :thinking:

**Comment by anubha-v-ardhan** [comment]
> @anubha-v-ardhan I have signed the cla. It still shows that not passed. What should I do?

: D It takes some time to update. Look its `yes` now. Hurry

**Review by anubha-v-ardhan — APPROVED** [review]
LGTM

**Comment by anubha-v-ardhan** [comment]
@kaiwalyakoparkar Can you squash all the commits into one commit and change the base to `dev-1.22-hi.1`.
That should make this PR ready to merge.

**Comment by kaiwalyakoparkar** [comment]
@anubha-v-ardhan Changes done as you asked for
-[ ] PR is open now
-[ ] Squashed all commits to one
-[ ] Changed the base to `dev-i.22-hi.1`

**Comment by anubha-v-ardhan** [comment]
@kaiwalyakoparkar Seems like something went wrong and a few other commits commits not related to this PR have crept in. Can you fix it.

**Comment by kaiwalyakoparkar** [comment]
@anubha-v-ardhan Oh... I will fix it soon

**Comment by kaiwalyakoparkar** [comment]
@anubha-v-ardhan could you please check it now?

**Comment by anubha-v-ardhan** [comment]
The extra file changes are gone but the commits are still there, you can use `git reset --hard HEAD~<no. of commits>` and the force push using `git push --force` to remove unwanted commits.

**Comment by kaiwalyakoparkar** [comment]
> The extra file changes are gone but the commits are still there, you can use `git reset --hard HEAD~<no. of commits>` and then force push using `git push --force` to remove unwanted commits.

Strange, I squashed them again :thinking:

**Comment by anubha-v-ardhan** [comment]
You need to rebase it too i guess

**Comment by kaiwalyakoparkar** [comment]
> You need to rebase it too i guess

Umm.. I tried a couple of things but that doesn't seem to work. Do you have any suggestions on how I can do that?
At my end, the commit log looks like this

**Comment by verma-kunal** [comment]
> > You need to rebase it too i guess
> 
> Umm.. I tried a couple of things but that doesn't seem to work. Do you have any suggestions on how I can do that?
> At my end, the commit log looks like this
> 

Hey @kaiwalyakoparkar , this is the last commit in this branch, right?

**Comment by kaiwalyakoparkar** [comment]
> > > You need to rebase it too i guess
> > 
> > 
> > Umm.. I tried a couple of things but that doesn't seem to work. Do you have any suggestions on how I can do that?
> > At my end, the commit log looks like this
> > 
> 
> Hey @kaiwalyakoparkar , this is the last commit in this branch, right?

I hope it is

**Comment by verma-kunal** [comment]
> > > > You need to rebase it too i guess
> > > 
> > > 
> > > Umm.. I tried a couple of things but that doesn't seem to work. Do you have any suggestions on how I can do that?
> > > At my end, the commit log looks like this
> > > 
> > 
> > 
> > Hey @kaiwalyakoparkar , this is the last commit in this branch, right?
> 
> I hope it is

I followed this approach:
- [ ] git rebase -i <commit id of the last commit (you want all commits above this to be squashed)>
- [ ] replacing all the 'pick' with 's' option (except the 1st one, in which all the commit would be merged)
- [ ] esc + `:x`
- [ ] write a final commit message!!
- [ ] esc + `:x`
- [ ] git push --force-with-lease origin <your-branch-name>

did you this in this way?

**Comment by verma-kunal** [comment]
i think so too!! this was your last commit
so, commit id below this one should be good iguess

Screenshot 2021-09-13 at 3 22 58 PM

**Comment by kaiwalyakoparkar** [comment]
> * [ ]  git rebase -i <commit id of the last commit (you want all commits above this to be squashed)>
> * [ ]  replacing all the 'pick' with 's' option (except the 1st one, in which all the commit would be merged)
> * [ ]  esc + `:x`
> * [ ]  write a final commit message!!
> * [ ]  esc + `:x`
> * [ ]  git push --force-with-lease origin `your-branch-name`
> 
> did you this in this way?

I will try this

**Comment by kaiwalyakoparkar** [comment]
@anubha-v-ardhan I tried a couple of techniques. Can you squash them while merging?

**Comment by verma-kunal** [comment]
> @anubha-v-ardhan I tried a couple of techniques. Can you squash them while merging?

@anubha-v-ardhan , i think there might have been some error while squashing!! So maybe we cab use the `label`, which was suggested by @sftim 

What do you think?

**Comment by anubha-v-ardhan** [comment]
the squash was successfull but few commits here are not related to this PR, @kaiwalyakoparkar Did you fetch and merged `kubernetes:main` to `kaiwalyakoparkar:docs/tasks/tools/included/_index.md`?

**Comment by sftim** [comment]
> commit id of the last commit

You can use your shell and Git to work out the right place to rebase automatically:
```bash
git rebase -i "$( git merge-base HEAD dev-1.22-hi.1 )"
```

To _fix_ this PR, do that and also change `pick` to `drop` for all the unwanted commits.

**Comment by anubha-v-ardhan** [comment]
The commits are not related to this PR and shouldn't be here.
@kaiwalyakoparkar Did you rebased against local `main` instead of `dev-1.22-hi.1`?

**Comment by kaiwalyakoparkar** [comment]
> The commits are not related to this PR and shouldn't be here.
> @kaiwalyakoparkar Did you rebase against local `main` instead of `dev-1.22-hi.1`?

Unfortunately yes

**Comment by verma-kunal** [comment]
> The commits are not related to this PR and shouldn't be here.
> @kaiwalyakoparkar Did you rebase against local `main` instead of `dev-1.22-hi.1`?

🤔 even i don't remember changing anything like this!!

**Comment by kaiwalyakoparkar** [comment]
Closing this PR to open a new one due to rebasing issues
cc: @sftim @anubha-v-ardhan

**Comment by anubha-v-ardhan** [comment]
Thank you, Do tag the tracking issue in new PR.

**Comment by verma-kunal** [comment]
here is the tracking issue: https://github.com/kubernetes/website/issues/29353

**Pull request closed without merging**